src/lib/holocene/navigation/drawer.svelte

```diff
@@ -28,8 +29,10 @@
   on:click-outside={onClose}
 >
   <div class="relative">
-    <div class="absolute right-0 top-0 cursor-pointer" on:click={onClose}>
-      <Icon name="close" />
+    <div class="absolute right-0 top-0">
+      <IconButton on:click={onClose}>
```

Comment:
Could this be shortened to ⬇️ ?
```suggestion
      <IconButton on:click={onClose} icon="close" />
```